nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/test/java/org/apache/nifi/provenance/TestPersistentProvenanceRepository.java

```diff
@@ -481,6 +485,165 @@ public void testIndexAndCompressOnRolloverAndSubsequentSearch() throws IOExcepti
         assertTrue(newRecordSet.getMatchingEvents().isEmpty());
     }
 
+    // TODO: Switch to 10,000.
```

Comment:
Whoops! Overlooked that. Was just a comment that I added in so that if i forgot to change the timeout back to 10,000 millis, the reviewer would catch it - glad it worked! :) Made it much longer so that i could set a breakpoint. Will change back to 10,000 millis and update the PR.